h5c/vic/src/vic-webapp/e2e/app.po.ts

```diff
@@ -97,6 +100,12 @@ export class VicWebappPage {
     this.clickByCSS(this.iconVicShortcut);
   }
 
+  navigateToHostAndClusters() {
```

Comment:
Where is this function being used? I'm not finding it anywhere in the spec files

Reply:
Yes, in fact, this function is no more used.